hr_payroll_commission/data/hr.salary.rule.category.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
File name on xml data should have "_" instead of ".", only csv files should be names with ".", and for all files from 2 modules..